kanidm_book/src/security_hardening.md

```diff
@@ -39,8 +40,10 @@ possible risks.
 
 ### Should be readonly to running uid
 
-Files such as configurations should be readonly to this UID/GID. This is so that if an attacker is
-able to gain code execution, they are unable to modify the configuration to write or over-write
+TODO is the running UID the kanidm daemon?
```

Comment:
The user id running the kanidm daemon shouldn't be able to write to the configuration files.